common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -476,6 +477,40 @@ def test_create_enrollment_with_mode(self):
         self.assertTrue(is_active)
         self.assertEqual(course_mode, 'professional')
 
+    def test_enrollment_includes_expired_verified(self):
+        """With the right API key, request that expired course verifications are still returned. """
+        # Create a honor mode for a course.
+        CourseModeFactory.create(
+            course_id=self.course.id,
+            mode_slug=CourseMode.HONOR,
+            mode_display_name=CourseMode.HONOR,
+        )
+
+        # Create a verified mode for a course.
+        CourseModeFactory.create(
+            course_id=self.course.id,
+            mode_slug=CourseMode.VERIFIED,
+            mode_display_name=CourseMode.VERIFIED,
+            expiration_datetime='1970-01-01 05:00:00'
+        )
+
+        # Passes the include_expired parameter to the API call
+        v_response = self.client.get(
+            reverse('courseenrollmentdetails', kwargs={"course_id": unicode(self.course.id)}), {'include_expired': '1'}
+        )
+        v_data = json.loads(v_response.content)
+
+        # Ensure that both course modes are returned
+        self.assertEqual(len(v_data['course_modes']), 2)
+
+        # Omits the include_expired parameter from the API call
```

Comment:
@rlucioni As far as I know, if it is included at all, it will return the expired mode. I brought this up to @wedaly but he said that the only time it will ever be included is when the expired mode is expected, so this fact didn't matter.